code/src/layouts/root-layout/RootLayout.tsx

```diff
@@ -19,6 +20,7 @@ const RootLayout = () => {
           <Outlet />
         </Suspense>
       </AppBox>
+      <ScrollToTopButton />
```

Comment:
I suggest rendering it inside `App.tsx` next to `AppSnackbar` component if we want to have  to top button on every page regardless of layout it lives in

Reply:
Moved